models/dfx/dfx_v0.5_ethereum_trades.sql

```diff
@@ -0,0 +1,106 @@
+{{ config(
+    schema = 'dfx_v0.5_ethereum',
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "dfx_v0.5",
+                                \'["henrystats"]\') }}'
+    )
+}}
+
+
+{% set project_start_date = '2021-05-19' %}
+
+WITH 
+
+dexs as (
+        SELECT 
+            evt_block_time as block_time, 
+            trader as taker, 
+            '' as maker, 
+            originAmount as token_sold_amount_raw, 
+            targetAmount as token_bought_amount_raw, 
+            NULL as amount_usd, 
+            origin as token_sold_address, 
+            target as token_bought_address, 
+            contract_address as project_contract_address, 
+            evt_tx_hash as tx_hash, 
+            '' as trace_address, 
+            evt_index
+        FROM 
+        dfx_finance_ethereum.Curve_evt_Trade
+        WHERE evt_block_time > NOW() - Interval '1 Week'
```

Comment:
I am not sure if you want to add an incremental filter here, but it seems dexs currently only includes the most recent  1 week's trades.

Reply:
updated, thanks ser.